wire-ios-sync-engine/WireSyncEngine.xcodeproj/project.pbxproj

```diff
@@ -3199,6 +3199,7 @@
 /* Begin PBXShellScriptBuildPhase section */
 		0145AE9C2B11580B0097E3B8 /* Run Sourcery */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
question: what does this do?

Reply:
It tells Xcode to always run this phase. Otherwise Xcode will try to understand dependencies but fails and prints a warning.

<img width="587" alt="Screenshot 2024-03-27 at 11 46 55" src="https://github.com/wireapp/wire-ios/assets/15628584/32362963-c078-4cdb-b28b-002f5eb42820">